docs/cost_usage_report_generation.rst

```diff
@@ -0,0 +1,181 @@
+
+Example ``nise report`` usage
+=============================
+
+nise is a command line tool::
```

Comment:
Do we need to have the usage in both the README & the cost_usage_report_generation? If we do - is there a reason that they are different?

Reply:
The README contains the `yaml` usage and `report` usage options, whereas in the `cost_usage_report_generation` I only put the report options and in the `yaml_generation` I only have the yaml options (if they are different beyond that, def let me know. I might be going crazy looking at these files). I just wanted the example readme files to show whatever options are actually pertinent info to what the user is trying to do.

Reply:
Ahhhh! That makes sense. Thanks for the clarification!